src/agrcgeocoding/geocode.py

```diff
@@ -321,6 +329,8 @@ def __init__(self):
     parser.add_argument('output', type=str)
     parser.add_argument('--wkid', default=26912, type=int, action='store')
     parser.add_argument('--locators', default='all', type=str, action='store')
+    parser.add_argument('--pobox', default='false', type=str, action='store')
+    parser.add_argument('--acceptScore', default='70', type=int, action='store')
```

Comment:
```suggestion
    parser.add_argument('--wkid', default=DEFAULT_SPATIAL_REFERENCE, type=int, action='store')
    parser.add_argument('--locators', default=DEFAULT_LOCATOR_NAME, type=str, action='store')
    parser.add_argument('--pobox', default=DEFAULT_POBOX, type=str, action='store')
    parser.add_argument('--acceptScore', default=DEFAULT_ACCEPT_SCORE, type=int, action='store')
```

This would allow us to maintain the default values in a single location. Also fixes the type of the default value for accept score (was a string).